modules/images/webp-uploads/load.php

```diff
@@ -704,3 +704,71 @@ function webp_uploads_update_rest_attachment( WP_REST_Response $response, WP_Pos
 	return rest_ensure_response( $data );
 }
 add_filter( 'rest_prepare_attachment', 'webp_uploads_update_rest_attachment', 10, 3 );
+
+/**
+ * Returns the attachment sources array ordered by filesize.
+ *
+ * @since n.e.x.t
+ *
+ * @param int    $attachment_id The attachment ID.
+ * @param string $size          The attachment size.
+ *
```

Comment:
We don't use a blank line between params and the return tag, so let's remove it to be consistent with other functions.
```suggestion
```

Reply:
Updated